src/oss-find-squats-lib/MutateExtension.cs

```diff
@@ -43,9 +45,12 @@ public static class MutateExtension
         /// <summary>
         /// Common variations known uniquely for Nuget/C#. Excludes '.' because Nuget will match "Microsoft.CST.OAT." against "Microsoft.CST.OAT", giving false positives for the same package.
         /// </summary>
-        internal static IEnumerable<IMutator> NugetMutators { get; } = BaseMutators.Where(x => x is not UnicodeHomoglyphMutator and not SuffixMutator)
+        internal static IEnumerable<IMutator> NugetMutators { get; } = BaseMutators.Where(x => 
+                x is not UnicodeHomoglyphMutator and not SuffixMutator and not CloseLettersMutator and not DoubleHitMutator)
```

Comment:
Why remove the double hit mutator from nuget?

Reply:
We remove it as we are customizing it from the `BaseMutators` implementation. Similarly to how we were doing it for the `SuffixMutator`